app/models/cinstance.rb

```diff
@@ -131,6 +131,8 @@ def validate_plan_is_unique?
   # "live", in a way).
   scope :live, -> { where(:state => ['live', 'deprecated'])}
 
+  scope :active_since_time_ago, ->(activity_time) { where.has { first_daily_traffic_at >= activity_time.ago } }
```

Comment:
Same as https://github.com/3scale/porta/pull/17/files#r229412297

Reply:
Ok, let's talk about it there 👍